src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -69,58 +75,113 @@ class ErtConfig:
     user_config_file: str = "no_config"
     config_path: str = field(init=False)
 
-    def __post_init__(self):
-        self.config_path = (
-            os.path.dirname(os.path.abspath(self.user_config_file))
-            if self.user_config_file
-            else os.getcwd()
-        )
-
     @classmethod
     def from_file(
-        cls, user_config_file, use_new_parser: bool = USE_NEW_PARSER_BY_DEFAULT
+        cls,
+        user_config_file,
+        collected_errors: List[ErrorInfo] = None,
```

Comment:
I don't think it is a good idea to add this additional parameter. It seems to only be used in a test.

Reply:
When implementing the linter, this will be used I guess?